corehq/apps/integration/kyc/models.py

```diff
@@ -125,6 +125,32 @@ def get_kyc_users_by_ids(self, obj_ids):
                 for user_obj in CommCareCase.objects.get_cases(obj_ids, self.domain)
             ]
 
+    def get_api_field_to_user_data_map_values(self):
```

Comment:
This is nice. The same mechanism can be used to improve the connection settings custom headers.